mod/reportedcontent/classes/Elgg/ReportedContent/Notifications/CreateReportedContentNotificationHandler.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Elgg\ReportedContent\Notifications;
+
+use Elgg\Notifications\NotificationEventHandler;
+use Elgg\Database\QueryBuilder;
+
+class CreateReportedContentNotificationHandler extends NotificationEventHandler {
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getSubscriptions(): array {
+		$admins = elgg_get_entities([
+			'type' => 'user',
+			'limit' => false,
+			'batch' => true,
+			'metadata_name_value_pairs' => [
+				[
```

Comment:
you know you can write these pairs more readable?
`'admin' => 'yes'`

Reply:
not going to change this